webrender/src/renderer.rs

```diff
@@ -2344,6 +2422,34 @@ impl Renderer {
                     .bind(&mut self.device, projection, 0, &mut self.renderer_errors);
                 GPU_TAG_PRIM_BLEND
             }
+            BatchKind::Brush(brush_kind) => {
+                match brush_kind {
+                    BrushBatchKind::Image(target_kind) => {
+                        match target_kind {
+                            RenderTargetKind::Alpha => {
+                                self.brush_image_a8.bind(
```

Comment:
nit: could move the actual `bind` call outside of that match

Reply:
Done